foodalert/static/foodalert/js/base.js

```diff
@@ -31,6 +35,8 @@ window.vueData.routes = [
             { path: "subscribed", component: UnauthzTemplate },
             { path: "audit", component: UnauthzTemplate },
             { path: "welcome", component: WelcomeTemplate },
+            { path: "close", component: CloseTemplate },
+            { path: "need-permit", component: NeedPermit },
```

Comment:
Need to change CloseTemplate to UnauthzTemplate here. Sarthak may just go ahead and fix this.

Reply:
This is fixed the last commit